spec/factories/follower.rb

```diff
@@ -0,0 +1,12 @@
+require './lib/pivotal-tracker/follower'
+
+FactoryGirl.define do
+  factory :follower, class: PivotalTracker::Follower do
+    sequence(:id) { |n| n }
+    name          { Faker::Name.name }
+    email         { Faker::Internet.email }
+    initials      { name.split.map{|name| name[0] } }
+    username      { Faker::Internet.user_name }
+    kind          'kind'
```

Comment:
Working on this now.

@jasonnoble let me know if you have suggestions about google_kind in factories/google_attachement.rb researching that now...